frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -26,7 +26,7 @@ function getErrorElement(error) {
                       Please wait a few minutes and try again.
         </p>
       </div>
-    );
+  );
```

Comment:
Why was this moved? It seemed to be lined up with `resolve` before?

Reply:
Ohh this was fixed when reverting back to the synchronous version of `getErrorElement`.